_studio/shared/umc/codec/h265_dec/src/umc_h265_task_supplier.cpp

```diff
@@ -1799,15 +1799,18 @@ UMC::Status TaskSupplier_H265::AddOneFrame(UMC::MediaData * pSource)
             case NAL_UT_PPS:
                 {
                     UMC::Status umsRes = DecodeHeaders(nalUnit);
+
+                    int32_t nalIndex = pMediaDataEx->index;
+                    int32_t size = pMediaDataEx->offsets[nalIndex + 1] - pMediaDataEx->offsets[nalIndex];
                     if (umsRes != UMC::UMC_OK)
                     {
-                        if (umsRes == UMC::UMC_NTF_NEW_RESOLUTION ||
+                        if (umsRes == UMC::UMC_WRN_REPOSITION_INPROGRESS)
+                        {
+                            umsRes = UMC::UMC_OK;
+                        }
+                        else if (umsRes == UMC::UMC_NTF_NEW_RESOLUTION ||
                             (nut == NAL_UT_SPS && umsRes == UMC::UMC_ERR_INVALID_STREAM))
                         {
-
-                            int32_t nalIndex = pMediaDataEx->index;
-                            int32_t size = pMediaDataEx->offsets[nalIndex + 1] - pMediaDataEx->offsets[nalIndex];
```

Comment:
We don't need this change. Otherwise, we'll loose [MFX_WRN_VIDEO_PARAM_CHANGED] notification.

Reply:
Thanks. It's addressed in https://github.com/Intel-Media-SDK/MediaSDK/pull/324